### PR TITLE
Add metadata to organizations

### PR DESCRIPTION
**Pull request by dplumb94**
Based on #41

### CODE REVIEW

File: sdk/protos/pike_payload.proto [hunk 1]
```diff
@@ -56,10 +56,12 @@ message CreateOrganizationAction {
   string id = 1;
   string name = 2;
   string address = 3;
+  repeated KeyValueEntry metadata = 4;
```
Comment (agunde406) [inline sdk/protos/pike_payload.proto]: This also needs to be added to protocol::pike::payload.

File: sdk/protos/pike_payload.proto [hunk 2]
```diff
@@ -56,10 +56,12 @@ message CreateOrganizationAction {
...
 }
 
 message UpdateOrganizationAction {
   string id = 1;
   string name = 2;
   string address = 3;
+  repeated KeyValueEntry metadata = 4;
```
Comment (agunde406) [inline sdk/protos/pike_payload.proto]: This also needs to be added to protocol::pike::payload.

File: sdk/protos/pike_state.proto [hunk 3]
```diff
@@ -36,6 +36,7 @@ message Organization {
   string org_id = 1;
   string name = 2;
   string address = 3;
+  repeated KeyValueEntry metadata = 4;
```
Comment (agunde406) [inline sdk/protos/pike_state.proto]: This also needs to be added to protocol::pike::state.

File: sdk/protos/pike_payload.proto (hunk 1 above)
Comment (dplumb94) [reply sdk/protos/pike_payload.proto]: Updated payload and state to add metadata to orgs in a new commit

File: docs/source/pike_transaction_family.rst [hunk 4]
```diff
@@ -71,6 +71,7 @@ An organization has three fields:
 - id: a unique identifier for the organization
 - name: a user defined identifier for the organization
 - address: a physical address for the organization
+- metadata: A list of key value pairs describing data about the organization.
```
Comment (chenette) [inline docs/source/pike_transaction_family.rst]: Extremely minor nitpick: Should be consistent with the case and non-punctuation of the other entries (start with a lower-case a and remove the period).

Comment (dplumb94) [reply docs/source/pike_transaction_family.rst]: I will fix that! Good catch

**Pull request merged**